cmd/erda-server/bootstrap.yaml

```diff
@@ -288,6 +288,7 @@ dop: { }
 erda.core.dop.taskerror: { }
 erda.dop.qa.unittest: { }
 project-management-report: { }
+performance-measure: { }
```

Comment:
efficiency-measurement

Reply:
done, and adujsted other `performance` title name